emails/tests/utils_tests.py

```diff
@@ -177,11 +178,34 @@ def test_get_domains_from_settings(self):
 
 @override_settings(SITE_ORIGIN="https://test.com")
 class RemoveTrackers(TestCase):
-    def setUp(self):
-        self.expected_content = (
-            '<a href="https://test.com/faq">A link</a>\n'
-            + '<img src="https://test.com/faq">An image</img>'
+    url = "https://test.com/contains-tracker-warning/#"
+    hyperlink_simple = "https://open.tracker.com/foo/bar.html"
+    imagelink_simple = "https://open.tracker.com/foo/bar.jpg"
+    hyperlink_complex = "https://foo.open.tracker.com/foo/bar.html"
+    imagelink_complex = "https://bar.open.tracker.com/foo/bar.jpg"
+    hyperlink_tracker_in_tracker = (
+        "https://foo.open.tracker.com/foo/bar.html?src=trckr.com"
+    )
```

Comment:
👍 I like these as class-level variables. 

nit: I think we should use them more in the tests as well, such as changing

```python
content = (
            '<a href="https://foo.open.tracker.com/foo/bar.html">A link</a>\n'
            + '<img src="https://bar.open.tracker.com/foo/bar.jpg">An image</img>'
        )
```

to

```python
content = (
            f'<a href="{self.hyperlink_complex}">A link</a>\n'
            f'<img src="{self.imagelink_complex}">An image</img>'
        )
```

(string concatenation is automatic inside parenthesis, so the `+` is unneeded)